src/ethereum/ethEngine.ts

```diff
@@ -361,26 +361,33 @@ export class EthereumEngine extends CurrencyEngine<
                   switch (payload.method) {
                     case 'eth_sendTransaction':
                     case 'eth_signTransaction': {
-                      const params = asObject({
+                      const txParam = asObject({
+                        from: asString,
+                        to: asOptional(asString),
+                        data: asString,
```

Comment:
If these 3 aren't used in the function, do we need to clean them? I only ask because the Ethereum engine is used in a lot of places, and I don't want to break if some other coin fails to provide these.

Reply:
I added them because I'm passing txParam to multicastServers where they're expected for the gas estimation. I can choose not to clean them if that seems less risky?

Reply:
Ah, that's where they are being used! Very good.